layers/meta-resin-edison/recipes-kernel/linux/linux-yocto-edison.bbappend

```diff
@@ -42,3 +42,6 @@ RESIN_CONFIGS_DEPS[ip6tables_nat] = " \
     CONFIG_NF_CONNTRACK_IPV6=m \
     CONFIG_IP6_NF_IPTABLES=y \
     "
+
+RESIN_CONFIGS_append = " smsc95xx"
```

Comment:
This doesn't sound like a edison specific configuration. We could have eth over USB for other boards too, right? Maybe this change should reside in meta-resin.